src/char_mapping.rs

```diff
@@ -45,6 +94,27 @@ table! {
 }
 
 table! {
+    #[doc = "A SequentialMapGroup as used in char-to-glyph encoding formats 8 and 12."]
+    pub SequentialMapGroup {
+        start_char_code (u32), // startCharCode
+        end_char_code   (u32), // endCharCode
+        start_glyph_id  (u32), // startGlyphID
+    }
+}
+
+table! {
+    #[doc = "A char-to-glyph encoding in format 0."]
+    pub Encoding0 {
+        format      (u16), // format
+        length      (u16), // length
+        language    (u16), // language
+        glyph_ids (Vec<u8>) |_, tape| { // glyphIdArray
```

Comment:
Add a new line before this line and adjust the formatting of the three topmost fields so that `(u16)`s are aligned to the end of `language`.